### PR TITLE
Added function to convert 3d numpy array to nested pandas dataframe

### PR DESCRIPTION
**Pull request by vedazeren**
#### Reference Issues/PRs
Fixes: #298 

#### What does this implement/fix? Explain your changes.
Added function to convert 3d numpy array to nested pandas dataframe

#### Does your contribution introduce a new dependency? If yes, which one? 
No

### CODE REVIEW

**Comment by mloning** [comment]
Looks great!

File: sktime/utils/tests/test_data_container.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    assert isinstance(array, np.ndarray)
+
+
+def test_from_3d_numpy_to_nested():
+    """Test the from_3d_numpy_to_nested() function.
+    """
+
+    array = np.random.normal(size=(5, 12, 2))
+    nested = from_3d_numpy_to_nested(array)
+    assert isinstance(nested, pd.DataFrame)
```
Comment (mloning) [inline sktime/utils/tests/test_data_container.py:23]: I should have thought about that earlier, perhaps better to use: 
```suggestion
    assert is_nested_dataframe(nested)
```
and add `from sktime.utils.data_container import is_nested_dataframe` to the imports.

**Comment by mloning** [comment]
Thank you @vedazeren for your contribution! 🎉

**Comment by vedazeren** [comment]
Thanks @mloning , feels great to contribute!

**Pull request merged**